app/models/canonical/sync/ingredients.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Canonical
+  module Sync
+    class Ingredients < AssociationSyncer
+      private
+
+      def model_class
+        Canonical::Ingredient
+      end
+
+      def json_file_path
+        Rails.root.join('lib', 'tasks', 'canonical_models', 'canonical_ingredients.json')
+      end
+
+      def preserve_associations?
+        false
```

Comment:
Because the number of associated alchemical properties is capped at 4, preserving existing ones will prevent ones that are in the JSON data from being added and cause the whole thing to error out.